src/components/hero/_macro.njk

```diff
@@ -3,7 +3,7 @@
 {% from "components/icons/_macro.njk" import onsIcon %}
 
 {# style adjustments if dark or light skin #}
-{% if params.censusThemeDark is defined and params.censusThemeDark %}
+{% if params.variants is defined and 'dark' in params.variants %}
```

Comment:
Tests for `params.variants` need to be added; there is a good example of this type of test here: https://github.com/ONSdigital/design-system/blob/master/src/components/accordion/_macro.spec.js#L57